_docs-sources/foundations/github-app/setup.md

```diff
@@ -0,0 +1,24 @@
+# Setup
+
+To install the Gruntwork.io GitHub App in your organization follow these steps.
+
+1. Login to the Gruntwork Developer Portal and navigate to Account Settings.
+
+   <!-- TODO: Add screenshot -->
+
+1. Click Install App.
+
+   <!-- TODO: Add screenshot -->
```

Comment:
```suggestion
1. Follow instructions to start the App installation in the [Gruntwork Developer Portal's Account Settings](https://app.gruntwork.io/account?scroll_to=github-app).
```

Trying to keep this instructions minimal and useful even when the UI changes(which it might with support for multiple Orgs)